src/Smalot/PdfParser/PDFObject.php

```diff
@@ -274,18 +275,31 @@ public function getText(Page $page = null)
                         $args = preg_split('/\s/s', $command[self::COMMAND]);
                         $y = array_pop($args);
                         $x = array_pop($args);
-                        if (((float) $x <= 0) ||
-                            (false !== $current_position_td['y'] && (float) $y < (float) ($current_position_td['y']))
-                        ) {
+
+                        /*
+                         * When converting pdf into text some offsets between text should at least outputted as spaces.
+                         * But small move-text operations may be normal, see https://github.com/smalot/pdfparser/issues/201
+                         *
+                         * To find out what we are handling with, get the max char width for current font and check if offset is bigger than that
```

Comment:
Better would be: "what we are dealing with"

Reply:
thanks, you are right.